src/js/tracks/text-track-settings.js

```diff
@@ -165,119 +165,123 @@ function setSelectedOption(target, value) {
   target.selectedIndex = i;
 }
 
+// I know what you're thinking: "This should be a tagged template!". I thought the same thing
+// when I first took a look. However,
```

Comment:
Kidding, I think I decided to confirm that I was right and never went back and finished the comment.